tests/test-mongoc-collection.c

```diff
@@ -648,13 +697,22 @@ test_index_storage (void)
    bson_error_t error;
    bool r;
    bson_t keys;
-
-   mongoc_index_opt_init (&opt);
-   mongoc_index_opt_wt_init (&wt_opt);
+   const char *engine;
 
    client = mongoc_client_new (gTestUri);
    ASSERT (client);
 
+   /* Skip unless we are on WT */
+   engine = storage_engine(client);
+   if (strcmp(engine, "wiredtiger") != 0 &&
```

Comment:
This string is coming straight from the command line options passed to mongod.  I guess as long as we only test with 3.0.0-rc7 or later there's no reason to include "wiredtiger" because the db won't take it.  On earlier rc's, though, it's valid